File: src/main/java/de/maggiwuerze/xdccloader/events/channel/server/ChannelUpdateEvent.java
package de.maggiwuerze.xdccloader.events.channel.server;

import de.maggiwuerze.xdccloader.events.EntityUpdateEvent;
import de.maggiwuerze.xdccloader.model.entity.Channel;

public class ChannelUpdateEvent extends EntityUpdateEvent<Channel> {

	public ChannelUpdateEvent(Object source, Channel channel) {
		super(source, channel);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/channel/server/ChannelUpdateEvent.kt
package de.maggiwuerze.xdccwebloader.events.channel.server

import de.maggiwuerze.xdccwebloader.events.EntityUpdateEvent
import de.maggiwuerze.xdccwebloader.model.entity.Channel

class ChannelUpdateEvent(source: Any, channel: Channel) : EntityUpdateEvent<Channel>(source, channel)